plugin/src/buildWorkletString.ts

```diff
@@ -84,7 +86,7 @@ export function buildWorkletString(
 
   const transformed = transformSync(code, {
     plugins: [prependClosureVariablesIfNecessary(closureVariables)],
-    compact: !includeSourceMap,
+    compact: true,
```

Comment:
It's too late to think, please somebody confirm if we can leave `true` here.

Reply:
If I remember correctly, this change was required for proper source mapping for Hermes debugger. When we use `compact: true` then some debugger breakpoints will be shifted by some random number of lines.

Reply:
Turns out we already effectively use `compact: true` when running an app. I've just checked it and breakpoint seem to work fine after this change.